boxoffice/models/category.py

```diff
@@ -15,5 +15,18 @@ class Category(BaseScopedNameMixin, db.Model):
     seq = db.Column(db.Integer, nullable=False)
     item_collection = db.relationship(ItemCollection,
         backref=db.backref('categories', cascade='all, delete-orphan',
-            order_by=seq, collection_class=ordering_list('seq', count_from=1)))
+        order_by=seq, collection_class=ordering_list('seq', count_from=1)))
```

Comment:
Indent required since these are parameters to the backref, not to the relationship.